app/alerts.py

```diff
@@ -88,50 +91,108 @@ def build_alerts_geojson():
+    markers_layer = dl.MarkerClusterGroup(children=markers, id='sites_markers')
 
-    # We convert it into geojson format (not a dl.GeoJSON object yet) and return it
-    markers = dlx.dicts_to_geojson(markers)
+    return markers_layer
 
-    return markers
 
+# ------------------------------------------------------------------------------
+# Fire alerts
+# The following block is dedicated to fetching information about fire alerts and displaying them on the map
+
+# This function creates alerts-related elements such as alert_button, alert_markers
+def build_alerts_elements(value):
+
+    # Fetching alert status and reusable metadata
+    alert_metadata = build_live_alerts_metadata()
+    alert_lat = alert_metadata["lat"]
+    alert_lon = alert_metadata["lon"]
+    alert_id = str(alert_metadata["id"])
+
+    # Building the button that allows users to zoom towards the alert marker
+    if value == 0:
+        alert_button = dbc.Button(
+            children="Départ de feu, cliquez-ici !",
+            color="danger",
+            block=True,
+            id='alert_button'
+        )
+    else:
+        alert_button = ""
+
+    # Building alerts_markers objects and wraps them in a dl.LayerGroup object
+    if value == 0:
+        icon = {
+            "iconUrl": 'https://marsfireengineers.com/assets/images/resources/firedetection.png',
+            "iconSize": [40, 40],       # Size of the icon
+            "iconAnchor": [20, 20]      # Point of the icon which will correspond to marker's location
+            # "popupAnchor": [-3, -76]  # Point from which the popup should open relative to the iconAnchor
+        }
+        alerts_markers = [dl.Marker(
+            id="marker_{}".format(alert_id),   # Setting a unique id for each alerts_markers
+            position=(alert_lat, alert_lon),
+            icon=icon,
+            children=[dl.Popup(
+                [
+                    html.H2("Alerte détectée"),
+                    html.P("Coordonées : {}, {} ".format(alert_lat, alert_lon)),
+                    html.Button("Afficher les données de détection",
+                                id=("display_alert_frame_btn{}".format(alert_id)),  # Setting a unique btn id
+                                n_clicks=0,
+                                className="btn btn-danger")
+                ])])]
+        alerts_markers_layer = dl.LayerGroup(children=alerts_markers, id='alerts_markers')
+    else:
+        alerts_markers_layer = ""
 
-# Once we have the positions of cameras, we output another GeoJSON object gathering these locations
-def build_alerts_markers():
-    markers = dl.GeoJSON(data=get_camera_positions(),
-                         id='markers')
+    return alert_button, alerts_markers_layer
+
+
+# This function either triggers a zoom towards the alert point each time the alert button is clicked
+# and sets the default zoom and center params for map_object
+def define_map_zoom_center(n_clicks):
```

Comment:
une proposition de nom: zoom_on_alert() ou bien alert_zoom(), c'est plus explicite ou pas ?

Reply:
I think that we should keep this 'define', 'build', 'display' logic to maintain readability and unity among methods. I also find it suitable to mention `zoom` and `center` : the 2 params updated here

Reply:
yup, I hadn't read the callback yet so it makes more sense as you say to keep the `zoom` and `center` reference in the function name